src/asana/controller.py

```diff
@@ -32,6 +32,7 @@ def update_task(
     update_task_fields = asana_helpers.extract_task_fields_from_pull_request(
         pull_request
     )
+    logger.info(f"Updating task {task_id} with fields: {update_task_fields}")
```

Comment:
this will log in the form of:
```
{'field_name': 'xml/jsonified field body'}
```
fyi. Seems fine to me but it might be a lot of logs